lib/rubocop/cop/rspec/message_spies.rb

```diff
@@ -12,17 +12,22 @@ module RSpec
       #
       #   # bad
       #   expect(foo).to receive(:bar)
+      #   subject
       #
       #   # good
+      #   allow(foo).to receive(:bar) # or use instance_spy
+      #   subject
       #   expect(foo).to have_received(:bar)
       #
       # @example `EnforcedStyle: receive`
```

Comment:
Side note: I've run the cop against real-world-rspec with both settings:
`EnforcedStyle: receive`: 57497 files inspected, 1881 offences detected
`EnforcedStyle: have_received`: 57497 files inspected, 26290 offenses detected
That leads me to a thought that `to receive` is more popular, despite the cop being enabled by default and the default enforced style.

I would rather not shove the style that is my personal preference.
@bquorning, a question to you as a supporter of the current enforced style. Do you think it is a sufficient reason to change the default enforced style, or it's fine to keep the existing state of affairs?

Reply:
To recap, my preference for spies (`to_receive`) came after reading about [4-phase testing](https://thoughtbot.com/blog/four-phase-test), and after realizing that `expect … to_receive … and_return` is [confusing](https://github.com/rubocop/rubocop-rspec/issues/221).

I realize by the number of tickets that it’s a controversial style, and I don’t want to force controversial on people. But I do fear that if disabled, most people will never discover the cop.